frontend/packages/insights-plugin/src/components/InsightsPopup/style.scss

```diff
@@ -1,3 +1,3 @@
-.co-insights__no-rules {
-	color: var(--pf-global--Color--200);
+.co-insights__timestamp {
+  display: inline-block;
```

Comment:
Helps to organize the "Last refresh: " text and the Timestamp on the same line

Reply:
I think you probably want the `simple` prop for Timestamp so you don't need any special styles, although that will remove the UTC tooltip.